modules/server/src/main/scala/observe/server/tcs/TcsControllerEpicsCommon.scala

```diff
@@ -574,33 +613,44 @@ object TcsControllerEpicsCommon {
       case p: ScienceFold.Position => epicsSys.scienceFoldPosCmd.setScfold(encode(p))
     }
 
+    override def configMountPos[C](
+      subsystems: NonEmptySet[Subsystem],
+      current:    C,
+      tc:         TelescopeConfig,
+      l:          Lens[C, BaseEpicsTcsConfig]
+    ): List[WithDebug[C => F[C]]] = List(
+      tc.offsetA.flatMap(o =>
+        applyParam(
+          subsystems.contains(Subsystem.Mount),
+          (l ^|-> BaseEpicsTcsConfig.offset).get(current),
```

Comment:
BTW these operators are being deprecated in monocle 3, just a reminder for when we do the port